syncer/checkpoint.go

```diff
@@ -290,37 +293,36 @@ func (cp *RemoteCheckPoint) Clear(tctx *tcontext.Context) error {
 		return err
 	}
 
-	cp.globalPoint = newBinlogPoint(minCheckpoint, nil, minCheckpoint, nil)
-
+	cp.globalPoint = newBinlogPoint(binlog.NewLocation(cp.cfg.Flavor), binlog.NewLocation(cp.cfg.Flavor), nil, nil)
 	cp.points = make(map[string]map[string]*binlogPoint)
 
 	return nil
 }
 
 // SaveTablePoint implements CheckPoint.SaveTablePoint
-func (cp *RemoteCheckPoint) SaveTablePoint(sourceSchema, sourceTable string, pos mysql.Position, ti *model.TableInfo) {
+func (cp *RemoteCheckPoint) SaveTablePoint(sourceSchema, sourceTable string, point binlog.Location, ti *model.TableInfo) {
 	cp.Lock()
 	defer cp.Unlock()
-	cp.saveTablePoint(sourceSchema, sourceTable, pos, ti)
+	cp.saveTablePoint(sourceSchema, sourceTable, point, ti)
```

Comment:
Do we need to do `Clone` for the passed in `point`? it seems some of the callers do the `Clone`, but some don't. If so, ditto for other exported methods like `SaveGlobalPoint`.

Reply:
yes, I haven't unified the usage for `Clone`, maybe it's better to let the caller to decide whether to allow the function to update the value, if not allowed, can use `Clone`. 
I add a `Clone` here, maybe we can unified the usage later.

Reply:
OK, it's fine.